pymatgen/entries/entry.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+# Copyright (c) Pymatgen Development Team.
+# Distributed under the terms of the MIT License.
+
+"""
+This module implements equivalents of the basic ComputedEntry objects, which
+is the basic entity that can be used to perform many analyses. ComputedEntries
+contain calculated information, typically from VASP or other electronic
+structure codes. For example, ComputedEntries can be used as inputs for phase
+diagram analysis.
```

Comment:
@ayushsgupta , can you update the header for the new `entry.py` file to more accurately reflect what's in here?

Something like "This module implements the base Entry class, which is ....   Other Entry classes such as ComputedEntry and PDEntry should inherit from this class."